setvalidator/no_null_values.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package setvalidator
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/function"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+var _ validator.Set = noNullValuesValidator{}
+var _ function.SetParameterValidator = noNullValuesValidator{}
+
+type noNullValuesValidator struct{}
+
+func (v noNullValuesValidator) Description(_ context.Context) string {
+	return "null values are not permitted"
```

Comment:
nit: similar to above suggestion
```suggestion
	return "All values in the set must be configured"
```